octoprint_mrbeam/util/material_csv_parser.py

```diff
@@ -4,42 +4,33 @@
 MRBEAM = "Mr Beam II"
 MRB_DREAMCUT = "MrB II Dreamcut"
 MRB_DREAMCUT_S = "MrB II Dreamcut S"
+MRB_DREAMCUT_x = "MrB II Dreamcut x"
 
 MRB_READY = "MrB II Dreamcut Ready"  # not used yet
 MRB_DREAMCUT_NOT_VALIDATED = "Dreamcut (not validated)"  # not used yet
 MRB_DREAMCUT_S_NOT_VALIDATED = "Dreamcut S (not validated)"  # not used yet
 
 DEFAULT_LASER_MODEL = "0"
 LASER_MODEL_S = "S"
+LASER_MODEL_x = "x"
+
+VALID_VERSIONS = [MRBEAM, MRB_DREAMCUT, MRB_DREAMCUT_S, MRB_READY, MRB_DREAMCUT_x]
 
 
 def model_ids_to_csv_name(device_model_id, laser_model_id):
     convert = {
+        (device_info.MODEL_MRBEAM_2_DC_x, LASER_MODEL_x): MRB_DREAMCUT_x,
```

Comment:
Maybe we can get rid of this mapping and just use the laser head model ID for the CSV name excluding only the DC_R1 and DC_R2 model handling. But this is optional.

Reply:
I will not change it as this will change with the new material settings anyway